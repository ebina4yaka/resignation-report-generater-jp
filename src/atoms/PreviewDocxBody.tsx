import React, { ReactElement } from 'react'
import { createStyles, makeStyles } from '@material-ui/core/styles'
import { getReason, requestTicket } from '../libs/export/textTemplates'

type Props = {
  reason: string
  dateOfRetirement: string
}

const useStyles = makeStyles(
  createStyles({
    root: {
      textAlign: 'left',
    },
  })
)

export default function PreviewDocxBody(props: Props): ReactElement {
  const { reason, dateOfRetirement } = props
  const classes = useStyles()
  return (
    <div className={classes.root}>
      <div>{getReason(reason, dateOfRetirement)}</div>
      <div>{requestTicket}</div>
    </div>
  )
}
