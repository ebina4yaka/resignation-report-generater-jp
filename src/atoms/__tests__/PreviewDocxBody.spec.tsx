import React from 'react'
import renderer from 'react-test-renderer'
import PreviewDocxBody from '../PreviewDocxBody'

test('Snapshot test for PreviewDocxBody', () => {
  const tree = renderer
    .create(
      <PreviewDocxBody reason="一身上の都合" dateOfRetirement="2020年11月7日" />
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
