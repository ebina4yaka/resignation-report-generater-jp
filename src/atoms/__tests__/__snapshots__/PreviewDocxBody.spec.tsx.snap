// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for PreviewDocxBody 1`] = `
<div
  className="makeStyles-root-1"
>
  <div />
  <div />
</div>
`;
